tests/test_backend.py

```diff
@@ -58,7 +58,7 @@ class FakeSESConnection:
     outbox = []
 
     def __init__(self, *args, **kwargs):
```

Comment:
Could remove this altogether now?

Reply:
No, we can't. `django_ses/__init__.py:109` will instantiate a session with parameters (`self.create_session().client(...)`). Since we're making the connection itself with `FakeSESConnection`, we must support `kwargs`. If we were to remove this, `FakeSESConnection` will thrown an error because it doesn't take arguments.

Reply:
Gotcha, a comment like `"""Override `SESConnection.__init__ to skip session creation."""` would be nice.

Reply:
Done! 🙏